### ISSUE TITLE
ph show group/entry -> No such entry

### ISSUE THREAD

**Issue by laur89** [Evidlo/passhole#46]
ph ver 1.9.4, installed via pip.

Listing db works

    ph list
    Internet
    ├── 2shared

Yet displaying the contents fails:

```
ph show Internet/2shared
No such entry Internet/2shared
```

Operating on a database used with keepassxc 2.6.2

~/.config/passhole.ini:
```
[default]
default: True
database: /full/path/to/db.kdbx
```

**Comment by Evidlo** [comment]
Should be fixed with 1.9.5.

**Comment by laur89** [comment]
Fix confirmed, cheers.
